src/components/Settings/DatasourceSettings/index.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import ListItem from 'react-md/lib/Lists/ListItem';
+import List from 'react-md/lib/Lists/List';
+import plugins from '../../../data-sources/plugins';
+
+interface IDatasourceSettingsProps {
+  settings: IDataSourceContainer;
+}
+
+interface IDatasourceSettingsState { 
+  selectedIndex: number;
+}
+
+export default class DatasourceSettings extends React.Component<IDatasourceSettingsProps, IDatasourceSettingsState> {
```

Comment:
We use DataSources (both Upper) in other places in the project, please update this one to be DataSourceSettings